app/controllers/projects_controller.rb

```diff
@@ -4,12 +4,17 @@ def index
       project.tenant.visible_to_lenders == true &&
       project.current_amount_needed > 0
     end
-    @categories = Category.all
+    @categories = Category.select(:name).all
+
+    if request.xhr?
+      require 'pry' ; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Space found before semicolon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.